testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -768,6 +768,102 @@ def test_warning(self, system, request):
         with pytest.warns(match="Some of the additional"):
             request.getfixturevalue(system)
 
+    def test_dump_reader_units_attribute(self):
+        """Test that DumpReader has proper units defined"""
+        from MDAnalysis.coordinates.LAMMPS import DumpReader
```

Comment:
Here and elsewhere in this diff, I think we should probably just keep imports at the top level per the usual approach.